packages/components/src/styles/components/form/layout.scss

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// FORM LAYOUT COMPONENTS
+//
+
+// FORM
+
+// Default max-width for form sections & similar components
+$hds-form-section-max-width: 768px;
+
+.hds-form {
+  --hds-form-section-max-width: #{$hds-form-section-max-width};
+
+  display: flex;
+  flex-direction: column;
+  gap: 32px;
+
+  // Set a max-width for the ButtonSet when it's within an Hds::Form
+  // Q: Should this be done (max-width on ButtonSet) and if so should it be done here or in the ButtonSet component?
+  .hds-button-set {
+    max-width: var(--hds-form-section-max-width, #{$hds-form-section-max-width});
```

Comment:
[question] are you sure you need a fallback here? since it's nested inside the `.hds-form` tag, the `--hds-form-section-max-width` is always defined

Reply:
Let me double check if I need those. It may have been left over from an earlier iteration.

Reply:
I got rid of it, thanks